cli/integration-test/sample-test/.lsp/config.edn

```diff
@@ -1,3 +1,4 @@
 {:linters {:clj-kondo {:ns-exclude-regex ""
                        :async-custom-lint? false}}
+ :log-path "/tmp/clojure-lsp.integration.out"
```

Comment:
Hum, we may have issues with some downstream that build clojure-lsp and run integration tests like [nix](https://github.com/NixOS/nixpkgs/blob/master/pkgs/development/tools/misc/clojure-lsp/default.nix) that doesn't accept that kind of absolute path

Reply:
Maybe we can pass a relative path? like:
```suggestion
 :log-path "clojure-lsp.integration-test.out"
```

Reply:
👍 I'll try that to see if it works.

Reply:
Yes! I hope that works with nix 😅 , I don't see why that wouldn't work now